x-pack/platform/plugins/shared/streams/server/routes/internal/streams/processing/grok_suggestions_handler.ts

```diff
@@ -77,6 +77,13 @@ export const handleProcessingGrokSuggestions = async ({
       review_fields: JSON.stringify(params.body.review_fields),
     },
   });
+
+  if (!('toolCalls' in response) || response.toolCalls.length === 0) {
+    throw new Error(
+      'Failed to generate grok suggestions due to error with the AI generation, please try again later.'
```

Comment:
Why is the error message here different from after 9.2?

Reply:
Because in >9.2 the error is thrown by a generic function that might be re-used in different context, while here the user-facing error is directly this one. In >9.2 versions, this error will be more generic, but still caught and re-thrown with a more specific message like this.